### PR TITLE
NAS-109601 / 21.04 / Move consolemsg to system.general

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/system.py [hunk 1]
```diff
@@ -1332,6 +1344,7 @@ async def ui_certificate_choices(self):
             Str('timezone', empty=False),
             Bool('crash_reporting', null=True),
             Bool('usage_collection', null=True),
+            Bool('consolemsg'),
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py:1347]: Can we use ui_consolemsg?

File: src/middlewared/middlewared/plugins/system.py [hunk 2]
```diff
@@ -240,9 +240,17 @@ async def do_update(self, data):
         hardware.
 
         When `syslogserver` is defined, logs of `sysloglevel` or above are sent.
+
+        `consolemsg` is a deprecated attribute and will be removed in further releases. Please, use `consolemsg`
+        attribute in the `system.general` plugin.
         """
+        consolemsg = None
+        if 'consolemsg' in data:
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py:248]: Wonder if it makes sense to generate a DeprecateWarning

Comment (themylogin) [reply src/middlewared/middlewared/plugins/system.py:248]: To the middleware log? What for, if it is a client issue?

Comment (william-gr) [reply src/middlewared/middlewared/plugins/system.py:248]: We do that elsewhere. Developers may look at our logs. It sounds better than nothing.

**Review by william-gr — APPROVED** [review]
Thanks

**Pull request merged**